src/content/en/2020/markup.md

```diff
@@ -380,12 +363,12 @@ Accordingly, we looked at the number of `details` and `summary` elements and it
       </tr>
     </tbody>
   </table>
-<figcaption>{{ figure_link(caption="Adoption of the <code>details</code> and <code>summary</code> elements.", sheets_gid="1406534257", sql_file="pages_element_count_by_device.sql") }}</figcaption>
+  <figcaption>{{ figure_link(caption="Adoption of the <code>details</code> and <code>summary</code> elements.", sheets_gid="1406534257", sql_file="pages_element_count_by_device.sql") }}</figcaption>
 </figure>
 
 ### Probability of element use
 
-Taking another look at element popularity, how likely is it to find a certain element in the DOM of a page? Surely, `html`, `head`, `body` are present on every page (even though [their tags are all optional](https://meiert.com/en/blog/optional-html/)), making them common elements, but what other elements are to be found?
+Taking another look at element popularity, how likely is it to find a certain element in the DOM of a page? Sure, `html`, `head`, `body` are present on every HTML page (even though [these tags are all optional](https://meiert.com/en/blog/optional-html/), browsers will automatically add them), making them common elements, but what other elements are to be commonly found?
```

Comment:
(Works for me but now I wonder if the sentence gets a bit unwieldy. Even for my taste 😅)

Reply:
For me "surely" doesn't work here and "sure" is I think what you meant.

I also feel it's also important to realise we are analysing post-processed HTML (as we are using a JavaScript custom metic that works on the DOM, rather than looking at the HMTL returned). So even though they are optional, they will have been added for the purposes  of our analysis. I feel that's an important point to get across.

Reply:
I agree. I discovered that html, head, body are there even if not in the raw html. So showing their percent is inaccurate. They are probably not 100% due to other errors.

Reply:
Could be some pages are txt or XML or other things other than HTML. So they won’t have html, head and body even post-processing.

Reply:
I think the “sure” change is fine, but the additional sentence fragment makes the sentence quite long and, unwieldy. I think the old version works but if we want to add more to the sentence we may need to break it up. (I can make a suggestion later if need be.)

Reply:
I'm tired of arguing. Let's remove the offending clause

```suggestion
Taking another look at element popularity, how likely is it to find a certain element in the DOM of a page? Sure, `html`, `head`, `body` are present on every HTML page (even though [these tags are all optional](https://meiert.com/en/blog/optional-html/)), making them common elements, but what other elements are to be commonly found?
```